incubator/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/KafkaGrpcBinding.java

```diff
@@ -55,6 +61,24 @@ public int workers(
     public KafkaGrpcBindingContext supply(
         EngineContext context)
     {
-        return new KafkaGrpcBindingContext(config, context);
+        return new KafkaGrpcBindingContext(config, context, this::activate, this::deactivate);
+    }
+
+    private boolean activate(
+        long bindingId)
+    {
+        final AtomicInteger worker = workers.computeIfAbsent(bindingId, id -> new AtomicInteger(INITIAL_WORKER_COUNT));
```

Comment:
We can use no-arg constructor here, `new AtomicInteger()`, defaults to `0` as it is an integer, same as Java fields.